lighthouse-treemap/app/styles/treemap.css

```diff
@@ -7,4 +7,88 @@
 body {
   margin: 0;
   overflow-y: hidden;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.lh-text-dim {
+  color: grey;
+}
+
+.lh-main {
+  display: grid;
+  height: 100vh;
+  grid-template-rows: 0.1fr 1fr 0fr;
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+}
+
+.lh-header--section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 20px;
+}
+.lh-header--url {
+  font-weight: bold;
+}
+
+.panel--modals {
+  display: flex;
+}
+
+.panel--treemap {
+  margin: 10px;
+}
+
+.view-mode {
+  cursor: pointer;
+  border: solid 1px blue;
+  padding: 5px;
+  font-size: 16px;
+}
+.view-mode:first-child {
+  border-top-left-radius: 2px;
+  border-bottom-left-radius: 2px;
+}
+.view-mode:last-child {
+  border-top-right-radius: 2px;
+  border-bottom-right-radius: 2px;
+}
+.view-mode--active {
+  background-color: lightblue;
+}
+
+header {
+  margin: 5px;
+}
+
+.webtreemap-node {
+  cursor: pointer;
```

Comment:
i somewhat remember experimenting with zoom-in/zoom-out cursors. it'd probably be interesting, but obv not blocking.